### PR TITLE
#236 audio content transcribing command and automatic trigger

### PR DESCRIPTION
**Pull request by Latemus**
Added functionality for transcribing speech in media content by replying to it with '/tekstitä'-command. Added setting for automatic Telegram voice and video note message transcribing when such is sent to a chat (original message or forwarded message). Can transcribe audio and video files as well for at least most common file types. Audio transcribing is done with OpenAI Whisper api.

### CODE REVIEW

File: bobweb/bob/test_main.py [hunk 1]
```diff
@@ -429,3 +433,112 @@ def test_min_max_normalize_handles_scale_movement(self):
...
+        text = 'Mary'
+        limit = 1
+        expected_chunks = ['M', 'a', 'r', 'y']
+        actual_chunks = split_text(text, limit)
+        self.assertEqual(actual_chunks, expected_chunks)
+
+
+class TestPagination(TestCase):
+
+    def test_simple_cases_with_incresing_page_count(self):
```
Comment (taturl) [inline bobweb/bob/test_main.py:469]:
```suggestion
    def test_simple_cases_with_increasing_page_count(self):
```

File: bobweb/bob/test_audio_transcribing.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        with open('bobweb/bob/resources/test/telegram_voice_message_mock.ogg', "rb") as test_sound_file:
+            chat, user = init_chat_user()
+            chat_entity = database.get_chat(chat.id)
+            chat_entity.voice_msg_to_text_enabled = True
+            chat_entity.save()
+
+            voice: Voice = create_mock_voice(chat.bot, test_sound_file)
+            user.send_voice(voice)
+
+            # Is expecte
```
Comment (taturl) [inline bobweb/bob/test_audio_transcribing.py:59]:
```suggestion
```

File: bobweb/bob/resources/bob_constants.py [hunk 3]
```diff
@@ -15,3 +15,6 @@
 DEFAULT_TIMEZONE = 'Europe/Helsinki'  # Default timezone for bot
 fitz = pytz.timezone(DEFAULT_TIMEZONE)
 utctz = pytz.UTC
+
+# Single telegram text message max content length in characters
+TELEGRAM_MESSAGE_MAX_LENGTH = 4096
```
Comment (taturl) [inline bobweb/bob/resources/bob_constants.py:20]: Where is this used?

File: bobweb/bob/activities/common_activity_states.py
[the remark this answers is not in the document]
Comment (taturl) [reply bobweb/bob/activities/common_activity_states.py:1]: Hmm, ehkä pois tai johonkin branchiin vaan talteen?

**Review by taturl — CHANGES_REQUESTED** [review]
Kommentit voisi resolvaa

File: bobweb/bob/resources/bob_constants.py (hunk 3 above)
Comment (Latemus) [reply bobweb/bob/resources/bob_constants.py]: Removed with all other non used code.

**Comment by Latemus** [comment]
> Kommentit voisi resolvaa

**Comment by Latemus** [comment]
@taturl valmis uudelleenarviointia varten. Kaikki korjaukset viimeisimmässä commitissa.

**Review by taturl — APPROVED** [review]

**Pull request merged**